src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md

```diff
@@ -11,25 +11,24 @@ All the available models are listed in the column `Model name`.
-| RegNet Y600                   | regnetY600         | ImageNet    | 224x224    | 76.18   | 92.34   | **0.77ms**                 | **1.22ms**                          | **2.64ms**                               | **2.93ms**                                        |           **2.39ms**            |
-| RegNet Y800                   | regnetY800         | ImageNet    | 224x224    | 77.07   | 93.26   | **0.74ms**                 | **1.19ms**                          | **2.77ms**                               | **3.04ms**                                        |           **2.81ms**            |
-| ResNet 18                     | resnet18           | ImageNet    | 224x224    | 70.6    | 89.64   | **0.52ms**                 | **0.95ms**                          | **2.01ms**                               | **2.30ms**                                        |           **4.56ms**            |
-| ResNet 34                     | resnet34           | ImageNet    | 224x224    | 74.13   | 91.7    | **0.92ms**                 | **1.34ms**                          | **3.57ms**                               | **3.87ms**                                        |           **7.64ms**            |
-| ResNet 50                     | resnet50           | ImageNet    | 224x224    | 81.91   | 93.0    | **1.03ms**                 | **1.44ms**                          | **4.78ms**                               | **5.10ms**                                        |           **9.25ms**            |
-| MobileNet V3_large-150 epochs | < WHY KEEP THIS?>  | ImageNet    | 224x224    | 73.79   | 91.54   | **0.67ms**                 | **1.11ms**                          | **2.42ms**                               | **2.71ms**                                        |           **1.76ms**            |
-| MobileNet V3_large-300 epochs | mobilenet_v3_large | ImageNet    | 224x224    | 74.52   | 91.92   | **0.67ms**                 | **1.11ms**                          | **2.42ms**                               | **2.71ms**                                        |           **1.76ms**            |
-| MobileNet V3_small            | mobilenet_v3_small | ImageNet    | 224x224    | 67.45   | 87.47   | **0.55ms**                 | **0.96ms**                          | **2.01ms** *                             | **2.35ms**                                        |           **1.06ms**            |
-| MobileNet V2_w1               | mobilenet_v2 ?     | ImageNet    | 224x224    | 73.08   | 91.1    | **0.46 ms**                | **0.89ms**                          | **1.65ms** *                             | **1.90ms**                                        |           **1.56ms**            |
+beit_base_patch16_224_imagenet.pth
```

Comment:
think this slipped out.

Reply:
changed in dcde517e71600e4183edb385da5d897b30830e4c